src/cli/deploy.ts

```diff
@@ -117,33 +146,10 @@ export default function (program: RootCmd): void {
         return;
       }
 
-      (await getUserConfirmation(opts)) || process.exit(0);
-
-      const builtModule = await buildModule();
-      if (!builtModule) {
-        return;
+      if (!(await getUserConfirmation(opts))) {
+        process.exit(0);
```

Comment:
```suggestion
        return;
```

This will not stop the PR from going through (technically it is a different story where we talk about this), but do we want to return instead instead of process exit when opts.confirm is negative?

Reply:
I agree that we should prefer to avoid `process.exit(0)` calls in favor of `return`. This PR focuses on just the peerDeps so I wanted to stay focused on that one task. Good callout for #1380!